components/molecule/avatar/test/index.test.js

```diff
@@ -192,7 +192,7 @@ describe(json.name, () => {
       const {getByText} = setup(props)
 
       // Then
-      expect(getByText('JS')).to.be.visible
+      expect(getByText('js')).to.be.visible
```

Comment:
The thing is I've changed the way initials are "uppercased", using the CSS transform property instead of the `.toUpperCase()`  js method. The final visual result is the same :)

Reply:
oh gotcha!